### PR TITLE
[Merged by Bors] - style: manually remove some leading `by`

### PR DESCRIPTION
**Pull request by grunweg**
The starting point for these changes was the automatic formatter, but all of these required manual changes,
to make the resulting line shorter than 100 characters. I do like the final result, though.

After this PR, there are 31 remaining occurrences, which I don't find obvious to fix.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/LinearAlgebra/Determinant.lean [hunk 1]
```diff
@@ -448,14 +448,13 @@ def LinearEquiv.ofIsUnitDet {f : M →ₗ[R] M'} {v : Basis ι R M} {v' : Basis
   map_smul' := f.map_smul
   invFun := toLin v' v (toMatrix v v' f)⁻¹
   left_inv x :=
-    calc
-      toLin v' v (toMatrix v v' f)⁻¹ (f x) = toLin v v ((toMatrix v v' f)⁻¹ * toMatrix v v' f) x :=
-        by rw [toLin_mul v v' v, toLin_toMatrix, LinearMap.comp_apply]
+    calc toLin v' v (toMatrix v v' f)⁻¹ (f x)
+      _  = toLin v v ((toMatrix v v' f)⁻¹ * toMatrix v v' f) x := by
```
Comment (sgouezel) [inline Mathlib/LinearAlgebra/Determinant.lean:452]:
```suggestion
      _ = toLin v v ((toMatrix v v' f)⁻¹ * toMatrix v v' f) x := by
```

File: Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean [hunk 2]
```diff
@@ -457,9 +457,9 @@ theorem map_biInf_comap {ι β} {I : Set ι} (hI : I.Nonempty) {f : α → β} (
 
 theorem restrict_iInf_restrict {ι} (s : Set α) (m : ι → OuterMeasure α) :
     restrict s (⨅ i, restrict s (m i)) = restrict s (⨅ i, m i) :=
-  calc
-    restrict s (⨅ i, restrict s (m i)) = restrict (range ((↑) : s → α)) (⨅ i, restrict s (m i)) :=
-      by rw [Subtype.range_coe]
+  calc restrict s (⨅ i, restrict s (m i))
+    _ = restrict (range ((↑) : s → α)) (⨅ i, restrict s (m i)) := by
+      rw [Subtype.range_coe]
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean:462]: Doesn't it fit on a line?

**Review by sgouezel — COMMENTED** [review]
bors d+

File: Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean (hunk 2 above)
Comment (grunweg) [reply Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean]: Indeed, good point. Changed.

**Comment by grunweg** [comment]
Thank you for the fast review.
bors r+

**Pull request closed without merging**